File: API Buena/SistemaEsmeralda/SistemaEsmeralda.Common/Models/CategoriaViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemaEsmeralda.Common.Models
{
   public class CategoriaViewModel
    {
        public int Usua_Id { get; set; }
        public int Cate_Id { get; set; }
        public string Cate_Categoria { get; set; }
        public int? Cate_UsuarioCreacion { get; set; }
        public DateTime? Cate_FechaCreacion { get; set; }
        public int? Cate_UsuarioModificacion { get; set; }
        public DateTime? Cate_FechaModificacion { get; set; }
        public bool? Cate_Estado { get; set; }
        [NotMapped]
        public string UsuarioCreacion { get; set; }

        [NotMapped]
        public string UsuarioModificacion { get; set; }


    }
}

File: sakai-ng-master/src/app/Models/CategoriaViewModel.ts
export class Categoria{
    cate_Id?:String;
    cate_Categoria?:String;
}
export class CategoriaEnviar {
    Usua_ID?: String;
    Usua_Id?: Number;
    Cate_Id: string;
    Cate_Categoria: string;
}

export class dropCategoria{
    value?:String;
    text?:String;
}

export class Fill {
    cate_Id: string;
    cate_Categoria: string;
    usuarioCreacion: string;
    usuarioModificacion: string;
    fechaCreacion : string;
    fechaModificacion : string;
}